index.html

```diff
@@ -1,270 +1,306 @@
+            <ul class="social-icons">
+              <li>
+                <i class="fa-brands fa-twitter"></i>
+              </li>
+              <li>
+                <i class="fa-brands fa-linkedin-in"></i>
+              </li>
               <li>
-                <img src="./assets/Javascript.png"/>
-                <p>JavaScript</p>
+                <i class="fa-solid fa-m"></i>
               </li>
               <li>
-                <img src="./assets/html.png"/>
-                <p>HTML</p>
+                <i class="fa-brands fa-github"></i>
               </li>
               <li>
-                <img src="./assets/css.png"/>
-                <p>CSS</p>
+                <i class="fa-brands fa-angellist"></i>
               </li>
             </ul>
+              <button >Get my resume</button>
           </div>
         </div>
-        <div class="frameworks frames">
-          <h4>Frameworks</h4>
-          <span>&gt;</span>
-        </div>
-        <div class="skills frames">
-          <h4>Skills</h4>
-          <span>&gt;</span>
-        </div>
+      
+      <ul class="my-about-skills-container">
+        <li class="my-about-skill-container">
+
+         <div class="Languages">
+                Languages</p>
+                <i class="fa-solid fa-angle-down"></i>
+            </div>
+        </li>
+        <li>
+          <ul class="my-about-skill-body-container">
+            <li class="my-about-skill-body">
+              <img
+                class="javascript-icon"
+                src="./assets/javascript.png"
+                alt="javascript-icon"title="JavaScript official icon"
+              />
+              <span class="my-about-skill-text">JavaScript</span>
+            </li>
+            <li class="my-about-skill-body">
+              <img
+                class="html-icon"
+                src="./assets/html.png"
+                alt="html-icon" title="Image title or description"
+              />
+              <span class="my-about-skill-text">&nbsp;HTML&nbsp;</span>
+            </li>
+            <li class="my-about-skill-body">
+              <img
+                class="css-icon"
+                src="./assets/css.png"title="html official language"
+                alt="css-icon"
+              />
+              <span class="my-about-skill-text">&nbsp;CSS&nbsp;</span>
+            </li>
+          </ul>
+        </li>
+        <li class="my-about-skill-container">
+          <div class="my-about-skill-heading">
+            <h4 class="my-about-heading-small">Frameworks</h4>
+            <i class="fa-solid fa-angle-right"></i>
+          </div>
+        </li>
+        <hr />
+        <li class="my-about-skill-container">
+          <div class="my-about-skill-heading">
+            <h4 class="my-about-heading-small">Skills</h4>
+            <i class="fa-solid fa-angle-right"></i>
+          </div>
+        </li>
+        <hr />
+     
+        </li>
+      </ul>
+          
       </div>
     </section>
```

Comment:
- [ ] Please note that this section is not divided into halves. The **About myself** section is 40% and the **Languages** section is 60%. In your design, the each occupy half the page. Please take a look at the Figma to confirm it.

**Figma**
![figma](https://github.com/the-faizmohammad/my-portfolio/assets/99656687/deae9d6f-8e8b-4d35-b9bf-5116d5887904)

**Your design**
![about](https://github.com/the-faizmohammad/my-portfolio/assets/99656687/d0882d78-8c01-4101-a235-a07a11fa6fab)